### DISCUSSION TITLE
Should I report an error or crash if it was automatically reported?

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#18106] [Q&A] [answered]
In Osu Lazer, it showed a notification that an error happened:
> An unhandled error has occurred.
> 
> This error has been automatically reported to the devs.

Should I report an error or a crash, if it was automatically reported already? Or is this just not worth the time, for me and the developers? 
I've already made a couple of posts here and I just want to help out. Thanks :)

**Comment by peppy** [comment, marked as answer]
Right now the automated reporting is turned off (but will be enabled soon). If you are seeing a crash that is important to have fixed to you, then please report it

**Reply by soopax** [reply]
Ah I see, thanks :)
